### DISCUSSION TITLE
Filtering stock of products (where it's > 0)

### DISCUSSION THREAD

**Discussion by kolbykruger** [swellstores/community#386] [Help] [answered]
Hi Swell dev team,

I was looking through the documentation on filtering products, and it looks like you can filter by price, category, and size. Is it possible to filter products where the stock is greater than 0?

**Comment by logeshswell** [comment, marked as answer]
Yes, it is possible to filter products where the stock > 0.

```ts
import swell from 'swell-js'

swell.init('<store-id>', '<public_key>');

const response = await swell.products.list({
  stock_status: 'in_stock'
});

// Alternate way
const response = await swell.products.list({
  stock_level: {
    $gt: 0
  },
});
```
